UploadDaemon/service/UploadDaemonService.xml

```diff
@@ -0,0 +1,33 @@
+<service>
+	<id>teamscale-dotnet-profiler-upload-daemon</id>
+	<name>Teamscale DotNet Profiler UploadDaemon</name>
+	<description>Teamscale DotNet Profiler UploadDaemon</description>
+
+	<!-- Adjust these paths to your setup. -->
+	<!-- PROFILER_DIR should be the path that contains Profiler32.dll -->
```

Comment:
Or Profiler64.dll?

Reply:
Yes, but they'll both be present always, so I didn't bother to include that ;-)